pool/app/pool_common/locales/i18n_de.ml

```diff
@@ -574,6 +574,7 @@ let confirmable_to_string confirmable =
    | DeleteMailing -> "den Versand", "löschen", None
    | DeleteMessageTemplate -> "das Nachrichtentemplate", "löschen", None
    | DeleteSession -> "die Session", "löschen", None
+   | DeleteSmtpServer -> "SMTP Server", "löschen", None
```

Comment:
@mabiede I'll need a hand with the right locale for "SMTP Server" here.

Reply:
I assume for the deleting messages, the way you've used the `Pool_common.Message.Field` is complete, so the changes for `DeleteSmtpServer` can be removed.